src/commands/rspec-test.yml

```diff
@@ -1,19 +1,23 @@
 description: "Test with RSpec. You have to add `gem 'rspec_junit_formatter'` to your Gemfile. Enable parallelism on CircleCI for faster testing."
 parameters:
+    include:
+        default: "spec/**/*_spec.rb"
```

Comment:
This introduced a bug - I think this needs to be double quoted.  See https://github.com/CircleCI-Public/ruby-orb/issues/86.